assets/components/contour/values.yaml

```diff
@@ -10,6 +10,7 @@ contour:
 envoy:
   image: docker.io/envoyproxy/envoy
   tag: v1.13.1
+  serviceType:
```

Comment:
just curious (helm ignorant here): is this empty to make it mandatory or something?

Reply:
Kind of. It's empty because we're setting the default in the component's Go code and I didn't want to set the default a second time in the values file. Regarding the Helm mechanism, AFAIK you can leave a key in `values.yaml` without a value and it's valid. If no value is provided at run time (via any of the supported Helm mechanisms), I guess you'll get an error when rendering the chart.

Reply:
Oh, okay. Thanks for the explanation! :)